commands/curation/curationaudit_test.go

```diff
@@ -398,30 +401,39 @@ func fillSyncedMap(pkgStatus []*PackageStatus) *sync.Map {
 	return &syncMap
 }
 
-func TestDoCurationAudit(t *testing.T) {
+func TestDoCurationAuditNpm(t *testing.T) {
```

Comment:
You removed the test for Maven, and made this name Npm (but the test cases also include the maven cases), is it mistake?...
if it is for all, make sure comments and var names also updated

Reply:
will rename